elastic/eui
# [React 18] Fix cross-version compatibility and unit test errors

## Summary

This PR fixes all remaining unit tests, adds cross-version compatibility to useRenderToText() and fixes code style in a few files I missed that were merged before (sorry!!)

## QA

1. Checkout this branch
2. Run `rm -rf node_modules && yarn`
3. Run `yarn lint`
4. Run `yarn test-unit`
5. Run `REACT_VERSION=17 yarn test-unit`
6. Confirm all of the above succeeded
